dev/com.ibm.ws.session.cache/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -15,6 +15,9 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
+httpSessionCache=HTTP Session Cache
+httpSessionCache.desc=Controls how HTTP sessions are persisted using JCache
```

Comment:
missing period at end of sentence.

Reply:
I fixed this locally, but it didn't seem to be included when I pushed the commit, I get it updated.